mypy.ini

```diff
@@ -3,32 +3,41 @@ packages = baybe
 
 ; at some point, these excludes should all be gone ...
 exclude = (?x)(
-          baybe/constraints
-          | baybe/recommenders
+          baybe/recommenders
           | baybe/searchspace
           | baybe/serialization
           | baybe/strategies
           | baybe/surrogates
-          | baybe/utils/basic.py
-          | baybe/utils/boolean.py
-          | baybe/utils/botorch_wrapper.py
-          | baybe/utils/chemistry.py
           | baybe/utils/dataframe.py
-          | baybe/utils/numeric.py
-          | baybe/utils/sampling_algorithms.py
           | baybe/acquisition.py
-          | baybe/campaign.py
           | baybe/deprecation.py
           | baybe/exceptions.py
           | baybe/objective.py
           | baybe/scaler.py
           | baybe/simulation.py
           | baybe/surrogate.py
-          | baybe/telemetry.py
           )
 
+[mypy-botorch.test_functions]
+ignore_missing_imports = True
+
 [mypy-setuptools_scm]
 ignore_missing_imports = True
 
 [mypy-scipy.spatial.distance]
-ignore_missing_imports = True
\ No newline at end of file
+ignore_missing_imports = True
+
+[mypy-sklearn.metrics]
```

Comment:
Could you just write one sentence why these things are necessary and what the error is? Not clear if this is what the two of you already discussed in your other conversation, but my understanding here is that this is basically that imports are missing in modules that we import? Or what does this mean?

Reply:
I think it's not needed to add any explanation here on our end since this is common mypy stuff. Mypy will by default complain about these packages, because it has no way to check its types, because the packages are not typed or they have no separate stub-files provided. So this is just the directive to tell mypy "OK acknowledged, just skip validation then". Since this is explained in depth in the mypy doc, no need for us to cover it (especially since not user-facing)